src/main/java/org/shanerx/backup/BackupMode.java

```diff
@@ -28,13 +28,15 @@ public class BackupMode {
     private int schedule;
     private int compressionLevel;
     private boolean recursive;
+    private int purgeAfter;
 
-    public BackupMode(String name, File dir, boolean allowManual, int schedule, int compressionLevel, boolean recursive) {
+    public BackupMode(String name, File dir, boolean allowManual, int schedule, int compressionLevel, boolean recursive, int purgeAfter) {
         this.name = name;
         this.dir = dir;
         this.allowManual = allowManual;
         this.schedule = Math.max(schedule, 0); // if <= 0 set to 0, otherwise to predefined value
         this.recursive = recursive;
+        this.purgeAfter = purgeAfter;
```

Comment:
Might to the same thing here as for the `schedule` field.

Reply:
Done!